deploy/olm-catalog/bundle/manifests/migration.openshift.io_migplans.yaml

```diff
@@ -206,6 +206,67 @@ spec:
               indirectImageMigration:
                 description: If set True, disables direct image migrations.
                 type: boolean
+              includedResources:
+                description: IncludedResources optional list of included resources in
+                  Velero Backup
+                items:
+                  description: GroupKind specifies a Group and a Kind, but does not
+                    force a version.  This is useful for identifying concepts during
+                    lookup stages without having partially valid types
+                  properties:
+                    group:
```

Comment:
@shawn-hurley JFYI, we are exposing this API. My understanding is this will be converted to Valero's groupResources field